amlip_cpp/include/amlip_cpp/node/ParentNode.hpp

```diff
@@ -21,6 +21,8 @@
 
 #include <memory>
 
+#include <fastdds/dds/domain/qos/DomainParticipantQos.hpp>
```

Comment:
This would be better to have it as a forward declaration, so no fastdds includes are done from these headers